docs/tutorials/infer_phys_pars.rst

```diff
@@ -0,0 +1,503 @@
+
+    plt.yscale('log')
+
+    plt.xlim(0., 1.)
+    plt.ylim(1., 1.e5)
+
+    plt.legend(loc='upper left')
+
+    plt.xlabel(r'fractional screen-pulsar distance $s$')
+    plt.ylabel(r'distance from Earth (pc)')
+
+    plt.show()
+
+This also shows that the effective distance sets a maximum on the distance to
+the screen :math:`d_\mathrm{s} < d_\mathrm{eff}`.
+
+
+Pulsar distance--orbital inclination relation
+---------------------------------------------
+
+The aplitude of the pulsar's orbital velocity signature :math:`A_\mathrm{p}`
+can be used to derive a relation between the distance to the pulsar system
+:math:`d_\mathrm{p}` and the sine of its orbital inclination
+:math:`\sin( i_\mathrm{p} )`, following
+
+.. math::
+
+    d_\mathrm{p} &= \frac{ \sqrt{ d_\mathrm{eff} } }{ A_\mathrm{p} }
+                    \frac{ K_\mathrm{p} }{ \sin( i_\mathrm{p} ) }
+                    b_\mathrm{p} \\
+                 &= \frac{ v_\mathrm{orb,E} K_\mathrm{p} }
+                         { A_\mathrm{E} A_\mathrm{p} }
+                    \frac{ b_\mathrm{E} b_\mathrm{p} }{ \sin( i_\mathrm{p} ) }.
+
+.. jupyter-execute::
+
+    nsini_p = 400
+    sini_p = np.arange(0.5/nsini_p, 1., 1./nsini_p)
+
+    b2_p = (1 - sini_p**2) / (1 - sini_p**2 * np.cos(chi_p)**2)
+    d_p = v_orb_e * k_p / (amp_e * amp_p) * np.sqrt(b2_e * b2_p) / sini_p
+
+.. jupyter-execute::
+
+    plt.figure(figsize=(7., 6.))
+
+    plt.plot(sini_p, d_p.to(u.pc))
+
+    plt.yscale('log')
+
+    plt.xlim(0., 1.)
+    plt.ylim(10., 1.e5)
+
+    plt.xlabel(r"sine of pulsar's orbital inclination $\sin( i_\mathrm{p} )$")
+    plt.ylabel(r"pulsar's distance from Earth $d_\mathrm{p}$ (pc)")
+
+    plt.show()
+
+
+The lens velocity
+-----------------
+
+Finally, it is possible to find a constraint on the projected lens velocity
+:math:`v_\mathrm{lens}`. This is best expressed in terms of some intermediate
+quantities derived above (:math:`\xi` and :math:`d_\mathrm{eff}`) and as a
+function the fractional screen--pulsar distance :math:`s`:
+
+.. math::
+
+    v_\mathrm{lens} = s \left( v_\mathrm{p,sys,eff}
+                               \pm \sqrt{ d_\mathrm{eff} } C \right),
+    \qquad \mathrm{with} \qquad
+    v_\mathrm{p,sys,eff} \simeq d_\mathrm{eff}
+                                \left[ \mu_{\alpha\ast} \sin( \xi )
+                                           + \mu_\delta \cos( \xi )
+                                \right].
+
+To compute a velocity from a proper motion and a distance, we use the
+:py:func:`~astropy.units.equivalencies.dimensionless_angles` equivalency. This
+takes care of handling the units of Astropy :py:class:`~astropy.units.Quantity`
+objects correctly when using the small-angle approximation
+(for further explanation, see the `Astropy documentation about equivalencies
+<https://docs.astropy.org/en/stable/units/equivalencies.html>`_).
+
+.. jupyter-execute::
+
+    s = [0., 1.]
+
+    v_p_sys_eff = ((d_eff * (mu_alpha_star * np.sin(xi) + mu_delta * np.cos(xi)))
```

Comment:
If going with all of the above, I'd write
```
mu_eff = psr_coord.pm_ra_cosdec * np.sin(xi) + psr_coord.pm_dec * np.cos(xi)
```
and then use `mu_eff` in the velocity equation.

Reply:
Implemented, but with ``mu_p_sys``.